docs/components/dartpad/dialogs/simple/main.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final SimpleDialog dialog = SimpleDialog(
+      title: Text('Set backup account'),
+      children: [
+        SimpleDialogItem(
+          icon: Icons.account_circle,
+          color: Colors.orange,
+          text: 'user01@gmail.com',
+          onPressed: () {
+            Navigator.pop(context, 'user01@gmail.com');
+          },
+        ),
+        SimpleDialogItem(
+          icon: Icons.account_circle,
+          color: Colors.green,
+          text: 'user02@gmail.com',
+          onPressed: () {
+            Navigator.pop(context, 'user02@gmail.com');
+          },
+        ),
+        SimpleDialogItem(
+          icon: Icons.add_circle,
+          color: Colors.grey,
+          text: 'Add account',
+          onPressed: () {
+            Navigator.pop(context, 'user02@gmail.com');
```

Comment:
why is this passing user02@gmail.com to pop?

Reply:
Oops, fixed